docs/release_notes/pending_release.rst

```diff
@@ -1,10 +1,17 @@
-SMQTK Pending Release Notes
+Pending Release Notes
 ===========================
```

Comment:
The `=` sequence should be shortened to match the character-length of the leading. Half style but also half functional: some tools will warn/complain when these don't line up.